### DISCUSSION TITLE
How can I change category and tag in blog page?

### DISCUSSION THREAD

**Discussion by erialc33** [MrGreensWorkshop/MrGreen-JekyllTheme#61] [How to] [answered]
Hello, how can I change category and tag in blog page? 
By the blog page I mean this: 

![image](https://user-images.githubusercontent.com/44615630/174061316-3b2da431-c40c-4336-970e-4abb0f413c80.png)

**Comment by MrGreensWorkshop** [comment, marked as answer]
Hi, check out the posts, its in there.
<img width="670" alt="Screen Shot 2022-06-16 at 21 26 22" src="https://user-images.githubusercontent.com/76992231/174069395-e3f65c9a-34b7-4279-bf66-f1d338eb3f4b.png">

see category and tags ?

**Reply by erialc33** [reply]
Heyya thanks! So I guess I just make the category and tags from the blogpost? 
I love your theme btw. Thank you so much.

**Reply by MrGreensWorkshop** [reply]
Yes, Liquid Code looks for definitions in blog posts and sorts them into categories, tags, and so on.
I'm glad you like it. You're welcome.
